modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -36,36 +36,88 @@ Establishing the right performance expectations refers to the fact that the real
 . Install Performance Addon Operator to the cluster.
 . Optional: Add a node to the {product-title} cluster.
 See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_for_real_time/8/html-single/tuning_guide/index#Setting_BIOS_parameters[Setting BIOS parameters].
-. Optional: Create a new machine config pool for real-time nodes.
+. Create a new machine config pool for real-time nodes:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-cnf
+  labels:
+    machineconfiguration.openshift.io/role: worker-cnf
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {
+           key: machineconfiguration.openshift.io/role,
+           operator: In,
+           values: [worker, worker-cnf],
+        }
+  paused: false
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-cnf: ""
+----
+
 . Add the node to the proper machine config pool, using node role labels.
 +
 You must decide which nodes will be configured with real-time workloads. It could be all of the nodes in the cluster or a subset of the nodes. The Performance Addon Operator expects all of the nodes are part of a dedicated machine config pool. If you use all of the nodes, you just point the Performance Addon Operator to the worker node role label. If you use a subset, you must group the nodes into a new machine config pool.
 
 . Create the `PerformanceProfile` with the proper set of housekeeping cores and `realTimeKernel: enabled: true`.
 
-. Specify a node selector in the `PerformanceProfile`, as shown here:
+. You can set `machineConfigPoolSelector` in `PerformanceProfile`:
 +
 [source,yaml]
 ----
 apiVersion: performance.openshift.io/v2
 kind: PerformanceProfile
 metadata:
- name: example-performanceprofile
+  name: performance
 spec:
-...
-  realTimeKernel:
-    enabled: true
+  cpu:
+    isolated: 1,3-39,41,43-79
+    reserved: 0,2,40,42
+  machineConfigPoolSelector:
+    machineconfiguration.openshift.io/role: worker-rt
   nodeSelector:
     node-role.kubernetes.io/worker-rt: ""
+  numa:
+    topologyPolicy: restricted
+  realTimeKernel:
+    enabled: true
 ----
```

Comment:
Lets use the same performance profile in the existing documentation and make few changes:

```
apiVersion: performance.openshift.io/v2
kind: PerformanceProfile
metadata:
 name: example-performanceprofile
spec:
...
  realTimeKernel:
    enabled: true
  nodeSelector:
     node-role.kubernetes.io/worker-rt: ""
  machineConfigPoolSelector:
     machineconfiguration.openshift.io/role: worker-rt
```